server/src/test/java/org/opensearch/index/SegmentReplicationStatsTrackerTests.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index;
+
+import org.opensearch.indices.IndicesService;
+import org.opensearch.test.OpenSearchTestCase;
+
+import static org.mockito.Mockito.mock;
+
+public class SegmentReplicationStatsTrackerTests extends OpenSearchTestCase {
+
+    private IndicesService indicesService = mock(IndicesService.class);
+
+    public void testRejectedCountWhenEmpty() {
```

Comment:
Do we need coverage for non-zero rejection count use case ?

Reply:
Added in new commit